db/large_data_handler.cc

```diff
@@ -95,7 +95,7 @@ future<> large_data_handler::maybe_delete_large_data_entries(sstables::shared_ss
         });
     }
     future<> large_cells = make_ready_future<>();
-    if (above_threshold(ldt::cell_size)) {
+    if (above_threshold(ldt::cell_size) || above_threshold(ldt::items_in_collection)) {
```

Comment:
This can be merged into the patch that adds the writing. This is really an artificial split.

Reply:
done in v5